### ISSUE TITLE
MDFTextAccessibility text contrast doesn't match Material spec

### ISSUE THREAD

**Issue by samnm** [material-components/material-components-ios#1447]
MDFTextAccessibility prefers black text for many more of the colors than is recommended in the [Material spec](https://material.io/guidelines/style/color.html#color-color-palette).

I've created a branch here to use MDFTextAccessibility for all the labels in the Palette example here:
https://github.com/samnm/material-components-ios/tree/palette-color-test

![screen shot 2017-05-19 at 12 04 52 pm](https://cloud.githubusercontent.com/assets/1251373/26256647/a84abac4-3c8b-11e7-9557-e1859c30d24a.png)

**Comment by ianegordon** [comment]
Closing as obsolete.
